locales/zh_CN/LC_MESSAGES/write_test.po

```diff
@@ -191,39 +256,58 @@ msgid ""
 "    foo()\n"
 "    assert r[None] == ti.approx(2)"
 msgstr ""
+"import taichi as ti\n"
+"\n"
+"# 将针对你这一端可用的所有后端进行测试\n"
+"@ti.test()\n"
+"def test_log10():\n"
+"    r = ti.var(ti.f32, ())\n"
+"\n"
+"    @ti.kernel\n"
+"    def foo():\n"
+"        r[None] = ti.log10(r[None])\n"
+"\n"
+"    r[None] = 100\n"
+"    foo()\n"
+"    assert r[None] == ti.approx(2)"
 
 #: ../../write_test.rst:123
 msgid ""
-"Simply using ``pytest.approx`` won't work well here, since it's tolerance"
-" won't vary among different Taichi backends. It'll be likely to fail on "
-"the OpenGL backend."
+"Simply using ``pytest.approx`` won't work well here, since it's tolerance won't "
+"vary among different Taichi backends. It'll be likely to fail on the OpenGL "
+"backend."
 msgstr ""
+"直接使用 ``pytest.approx`` 在这里表现不是很好，因为它的容忍度在不同的 Taichi 后"
```

Comment:
```suggestion
"直接使用 ``pytest.approx`` 在这里表现不是很好，因为它的公差在不同的 Taichi 后"
```

My bad. `Tolerance` here should be the mathematics meaning:

https://www.mathsisfun.com/definitions/tolerance.html